cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/service/ConcurrentStreamWriterTestBase.java

```diff
@@ -181,12 +185,14 @@ private boolean verifyEvents(int threads, int msgPerThread, List<StreamEvent> ev
     return true;
   }
 
-  private ConcurrentStreamWriter createStreamWriter(String streamName, StreamAdmin streamAdmin, int threads,
-                                                    StreamFileWriterFactory writerFactory) throws IOException {
+  private ConcurrentStreamWriter createStreamWriter(String streamName, String accountId, StreamAdmin streamAdmin,
+                                                    int threads, StreamFileWriterFactory writerFactory)
+    throws Exception {
     StreamConfig streamConfig = streamAdmin.getConfig(streamName);
     streamConfig.getLocation().mkdirs();
 
-    StreamMetaStore streamMetaStore = new NoOpStreamMetaStore();
+    StreamMetaStore streamMetaStore = new InMemoryStreamMetaStore();
+    streamMetaStore.addStream(accountId, streamName);
```

Comment:
It's the ``ConcurrentStreamWriter`` class itself that requires a store. It uses it to verify that a stream exists when we want to enqueue to a stream. It has that logic:

```
if (!streamMetaStore.streamExists(accountId, streamName)) {
        throw new IllegalArgumentException("Stream not exists");
      }
```